app/controllers/accounts_controller.rb

```diff
@@ -68,20 +68,27 @@ def get_profile
     if @current_user
       user = User.find_by(id: params[:id].to_i)
 
-      if not user
+      unless user
         render(inertia: 'NotFound')
         return
       end
 
+      user_json = user.as_json(only: [:id, :name, :username, :description, :avatar])
+
+      student_reviews = Review.where(connection_id: Connection.where(match_id: MatchRequest.where(student_id: user.id)))
```

Comment:
can't we do this on a single query using a join?

Reply:
This was the only way I could find at the time. I've been looking into it for a bit now but can't seem to figure it out, because the most nested query has a query parameter which I don't know how to pass to a join query in rails